server/mdm/scep/cryptoutil/cryptoutil_test.go

```diff
@@ -0,0 +1,54 @@
+package cryptoutil
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rsa"
+	"math/big"
+	"testing"
+)
+
+func TestGenerateSubjectKeyID(t *testing.T) {
+	t.Skip("skipping as this test has been failing in source repo and upstream for a while")
```

Comment:
@jahzielv the test is failing because of the made up `X` and `Y` points for the ECDSA key on line 19. This should fix it:

```diff
diff --git a/cryptoutil/cryptoutil_test.go b/cryptoutil/cryptoutil_test.go
index ab83c2e..afc2368 100644
--- a/cryptoutil/cryptoutil_test.go
+++ b/cryptoutil/cryptoutil_test.go
@@ -4,18 +4,24 @@ import (
        "crypto"
        "crypto/ecdsa"
        "crypto/elliptic"
+       "crypto/rand"
        "crypto/rsa"
        "math/big"
        "testing"
 )

 func TestGenerateSubjectKeyID(t *testing.T) {
+       ecdsaKey, err := ecdsa.GenerateKey(elliptic.P224(), rand.Reader)
+       if err != nil {
+               t.Fatal(err)
+       }
+
        for _, test := range []struct {
                testName string
                pub      crypto.PublicKey
        }{
                {"RSA", &rsa.PublicKey{N: big.NewInt(123), E: 65537}},
-               {"ECDSA", &ecdsa.PublicKey{X: big.NewInt(123), Y: big.NewInt(123), Curve: elliptic.P224()}},
+               {"ECDSA", &ecdsa.PublicKey{X: ecdsaKey.X, Y: ecdsaKey.Y, Curve: elliptic.P224()}},
        } {
                test := test
                t.Run(test.testName, func(t *testing.T) {
```

Reply:
Nice catch!